backend/bitboxbase/updater/updater.go

```diff
@@ -84,19 +91,35 @@ func (updater *Updater) Connect(ip string, bitboxBaseID string) error {
 		return errors.New("updater: Unexpected Response Body Bytes")
 	}
 	updater.running = true
-	go listenWebsocket(updater, ip, bitboxBaseID)
+	go listenWebsocket(updater, bitboxBaseID)
 	return nil
 }
 
+//GetEnv makes an http GET request to the base and on success returns some environment data from the base such as the electrs port
+func (updater *Updater) GetEnv() ([]byte, error) {
+	response, err := http.Get(updater.getEnvURL.String())
+	if err != nil {
+		updater.running = false
```

Comment:
call `updater.Stop()` so it is compatible with future changes.

Also, do you even need to stop? What if it is a transient error?